playbooks/roles/plugins/foreman_docker/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+- name: 'Prepare hammer taxonomy options'
+  set_fact:
+    foreman_docker_hammer_taxonomy_params: "--organizations '{{ foreman_docker_organization }}' --locations '{{ foreman_docker_location }}'"
+  when: foreman_docker_use_taxonomies
+
+- name: 'Prepare blank hammer taxonomy options'
+  set_fact:
+    foreman_docker_hammer_taxonomy_params: ""
+  when: not foreman_docker_use_taxonomies
+
+- name: "Install foreman docker plugin"
+  yum:
```

Comment:
Doesn't the installer have an option for this? There's at least a class for it in puppet-foreman.